### ISSUE TITLE
Want to change the code

### ISSUE THREAD

**Issue by KaChloe** [Jiaoyang-Li/CBSH2-RTC#8]
Hi,Jiaoyang-Li. Thank you for your great contribution! I've learned a lot from your work. If I want to have multiple targets per agent, and I want to access them in a specified order, do I need to make a big change to the code?

**Comment by Jiaoyang-Li** [comment]
Yes, this can be done. The modification to the basic version (i.e., CBS without improvement) should be easy. You just need to change the low-level search from A* to multi-label A*. But, if you want to keep those CBS improvements, it requires some additional efforts to take care of data structures like MDD. You can find some details in this paper: https://ieeexplore.ieee.org/document/9812020

**Comment by KaChloe** [comment]
Thank you very much！
